### PR TITLE
small note

### PR DESCRIPTION
**Pull request by joinimran**
As sign-in blocking is also applicable on room and equipment mailboxes.

Problem: https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/issues/770

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/email/create-a-shared-mailbox.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ The new admin center is available to all Microsoft 365 admins. You can opt in by
 
 4. In the **Edit sign-in status** flyout, verify that Block the user from signing in is selected, select **Save** and then **Close**.
 
-For instructions on how to block sign-in for accounts using Azure AD PowerShell (including many accounts at the same time), see [Block user accounts with Office 365 PowerShell](https://docs.microsoft.com/office365/enterprise/powershell/block-user-accounts-with-office-365-powershell).
+Sign in blocking can be used to block Room and Equipment mailboxes. For instructions on how to block sign-in for accounts using Azure AD PowerShell (including many accounts at the same time), see [Block user accounts with Office 365 PowerShell](https://docs.microsoft.com/office365/enterprise/powershell/block-user-accounts-with-office-365-powershell).
```
Comment (JohanFreelancer9) [inline Office365-Admin/email/create-a-shared-mailbox.md]:
```suggestion
Sign-in blocking can be used to block Room and Equipment mailboxes. For instructions on how to block sign-in for accounts using Azure AD PowerShell (including many accounts at the same time), see [Block user accounts with Office 365 PowerShell](https://docs.microsoft.com/office365/enterprise/powershell/block-user-accounts-with-office-365-powershell).
```

**Comment by joinimran** [comment]
Ready to go live.

**Comment by mypil** [comment]
@cmcatee-MSFT - PR has been copy edited and is ready for final review. This is based on issue #770.

Thank you.

cc: @kenwith

**Comment by joinimran** [comment]
@cmcatee-MSFT Sorry, I was not getting your point. You want me to remove the paragraph from this PR and moved it to https://docs.microsoft.com/en-us/office365/admin/manage/room-and-equipment-mailboxes?view=o365-worldwide. Am I correct?

**Comment by cmcatee-MSFT** [comment]
@joinimran Yes, that's correct. It doesn't belong in the shared mailboxes article. It's more appropriate for the rooms and equipment article.

**Comment by joinimran** [comment]
@cmcatee-MSFT I have created a new PR https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/pull/816 to add a note for the same and closing this PR as is.

**Pull request closed without merging**